tests/scr/testContainerInit

```diff
@@ -205,44 +205,20 @@ function wait_docker_ready() {
 }
 
 function install_sysbox_pkg() {
-
 	if [ ! -f ${SB_PACKAGE_FILE} ]; then
 		echo "Did not find sysbox installer: file ${SB_PACKAGE_FILE} does not exist"
 		exit 1
 	fi
 
-	set -e
-
+	# These quirks are required for apt-get to work correctly inside the test container
 	rm -rf /usr/sbin/policy-rc.d
+	dpkg -i dummy > /dev/null 2>&1 || true
```

Comment:
What is this for?

Reply:
These are quirks required for apt-get to work correctly inside the test container. The arise from the fact that we mount a host dir into the test container's `/var/run` directory, thus wiping out whatever apt & dpkg had stored in that dir in the test container image. These commands help to reinitialize things.